packages/template/env-spn

```diff
@@ -0,0 +1,4 @@
+VUE_APP_API_COSMOS=https://rest.alpha.starport.network
+VUE_APP_WS_TENDERMINT=wss://rpc.alpha.starport.network/websocket
+VUE_APP_API_TENDERMINT=https://rpc.alpha.starport.network
+VUE_APP_ADDRESS_PREFIX=cosmos
```

Comment:
Can we use `.env.dev` as a filename and `vue-cli-service build --mode dev`?

https://cli.vuejs.org/guide/mode-and-env.html#modes

Reply:
It's been fixed in the pending PR but since this is the released version, can we merge it to master as is and will finish up the PR for the next release